giraffe/src/types/index.ts

```diff
@@ -82,12 +82,15 @@ export type LineInterpolation =
 */
 export type LineHoverDimension = 'x' | 'y' | 'xy'
 
+export type LinePosition = 'overlaid' | 'stacked'
+
 export interface LineLayerConfig {
   type: 'line'
   x: string
   y: string
   fill?: string[]
   colors?: string[]
+  position?: LinePosition
```

Comment:
do you think it would make sense to provide a default here?

Reply:
Yes, makes perfect sense. I left it out by mistake. Will add that in with the next push.